.github/dependabot.yml

```diff
@@ -20,6 +20,12 @@ updates:
     schedule:
       interval: daily
       time: "00:00"
+  - package-ecosystem: gomod
```

Comment:
I don't like this. Dependabot keeps deps as close to edge as possible, the tool deps we use generate and result in actual code. Dependabot automation saves us having to remember to do it.